src/test/resources/com/google/jenkins/plugins/storage/integration/downloadStepPipeline.groovy

```diff
@@ -21,7 +21,8 @@ pipeline {
             steps{
                 step([$class: 'DownloadStep', credentialsId: env
                         .CREDENTIALS_ID,  bucketUri: "gs://${env.BUCKET}/${env.PATTERN}",
-                      localDirectory: "${env.DIR}/**/*"])
+                      localDirectory: "${env.DIR}"])
```

Comment:
`localDirectory` requires path relative to the workspace, so wildcards are treated as folders though.